src/prompts/text.rs

```diff
@@ -404,6 +416,9 @@ impl<'a> TextPrompt<'a> {
 
     fn prompt<B: TextBackend>(mut self, backend: &mut B) -> InquireResult<String> {
         let final_answer: String;
+        if !self.input.is_empty() {
+            self.update_suggestions();
+        }
```

Comment:
I understand this was necessary to have the correct suggestions loaded before the first render, but it looks a little bit out-of-place, I'll try to think of a way to refactor the Text prompt to process the state before the first render.

Reply:
> I understand this was necessary to have the correct suggestions loaded before the first render
Yes exactly